src/keeper/contracts/Dispenser.ts

```diff
@@ -9,10 +10,15 @@ export default class Dispenser extends ContractBase {
     }
 
     public async requestTokens(
-        amount: number | string,
+        amount: number | string | BigNumber,
         receiverAddress: string,
         params?: TxParameters
     ) {
-        return this.send('requestTokens', receiverAddress, [String(amount)], params)
+        return this.send(
+            'requestTokens',
+            receiverAddress,
+            [new BigNumber(amount)],
```

Comment:
should this be converted to a string also? Like here https://github.com/nevermined-io/sdk-js/pull/238/files#diff-ba42c89b863e0156bc2e1cec92a59d9591a8b8de1c638246ed4eb6cc30521f0eR112